### ISSUE TITLE
Fails to run when directory has `+` in its name

### ISSUE THREAD

**Issue by RealTheSunCat** [build-cpp/cmkr#61]
Hi! I love this tool, but I ran into an issue when someone tried to compile a program:

cmkr fails to run when the path to the directory it's in has a `+` in its name, with the following error:
```cmake
RegularExpression::compile(): Nested *?+.
RegularExpression::compile(): Error in compile.

CMake Error at cmkr.cmake:53 (if):
  if given arguments:

    "NOT" "CMKR_CACHED_EXECUTABLE" "STREQUAL" "CMKR_EXECUTABLE" "AND" "CMKR_EXECUTABLE" "MATCHES" "^/home/apm/Documents/C++/ReverseAbductionSimulator/build/_cmkr"

  Regular expression "^/home/apm/Documents/C++/ReverseAbductionSimulator/build/_cmkr" cannot compile
Call Stack (most recent call first):
  CMakeLists.txt:12 (include)


-- Configuring incomplete, errors occurred!
```

**Comment by mrexodia** [comment]
Thanks for the report! I’ll fix this soon, shouldn’t be too difficult.

**Comment by mrexodia** [comment]
This has been fixed in 0.2.14. For your project you can replace `cmkr.cmake` with the one [here](https://github.com/build-cpp/cmkr/blob/main/cmake/cmkr.cmake).
